lib/components/popups/course-search/postreqs/PostReqSection.tsx

```diff
@@ -0,0 +1,302 @@
+import React, { useState, useEffect, FC } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  selectSemester,
+  selectYear,
+  updateSearchStack,
+  selectVersion,
+  selectInspectedCourse,
+} from '../../../../slices/searchSlice';
+import { getCourse, getCourseYear } from '../../../../resources/assets';
+import {
+  selectCurrentPlanCourses,
+  selectPlan,
+} from '../../../../slices/currentPlanSlice';
+import { selectCourseCache } from '../../../../slices/userSlice';
+import { selectCourseToShow } from '../../../../slices/popupSlice';
+import {
+  SISRetrievedCourse,
+  UserCourse,
+  Year,
+} from '../../../../resources/commonTypes';
+
+const PostReqSection: FC = () => {
+  const dispatch = useDispatch();
+  const version = useSelector(selectVersion);
+  const inspected = useSelector(selectInspectedCourse);
+  const currPlanCourses = useSelector(selectCurrentPlanCourses);
+  const semester = useSelector(selectSemester);
+  const year = useSelector(selectYear);
+  const currentPlan = useSelector(selectPlan);
+  const courseCache = useSelector(selectCourseCache);
+  const courseToShow = useSelector(selectCourseToShow);
+
+  const convertTermToInt = (term: string): number => {
```

Comment:
lmao did not realize they were already alphabetically sorted, good to know!
Changed it to an enum anyway just in case any errors in semester type(tho ig its not needed?). Lmk if you think its better to just alphabetically.

Reply:
nah enum more maintainable